src/engine/plugins/tile.cpp

```diff
@@ -181,10 +230,46 @@ Status TilePlugin::HandleRequest(const api::TileParameters &parameters, std::str
     // This hits the OSRM StaticRTree
     const auto edges = facade.GetEdgesInBox(southwest, northeast);
 
-    std::vector<int> used_weights;
-    std::unordered_map<int, std::size_t> weight_offsets;
+    std::vector<int> used_line_ints;
```

Comment:
what is a `line int`? explanation of what this is supposed to be would be great/or a clearer name